### PR TITLE
add support for running in docker

### PR DESCRIPTION
**Pull request by dragonchaser**
This PR adds support for running LCARS inside a docker container.

### CODE REVIEW

**Comment by dragonchaser** [comment]
@louh can you take a look at this?

**Comment by louh** [comment]
@dragonchaser I apologize for the late response.

I gave this a try, following the README instructions. Here is what happens:

```
➜  lcars git:(dragonchaser-add-docker) docker build -t lcars:latest
ERROR: "docker buildx build" requires exactly 1 argument.
See 'docker buildx build --help'.

Usage:  docker buildx build [OPTIONS] PATH | URL | -

Start a build
```

I modified the build command slightly to add a `.` at the end like so:

```
docker build -t lcars:latest .
```

This appeared to work. Can you confirm if this period is required, and if so, update the README? thanks.

File: README.md [hunk 1]
```diff
@@ -5,3 +5,19 @@
 Many recreations of LCARS exist as static graphics or as Flash animations. This project shows that one can develop a responsive LCARS interface entirely with modern web technologies. The interface responds to fit any screen and collapses to a minimal form for mobile devices. Even works on [PADDs](https://memory-alpha.fandom.com/wiki/Personal_Access_Display_Device) (ahem, tablets)!
 
 **Questions, comments, or want to contribute?** [Open an issue](https://github.com/louh/lcars) or [message me on Twitter @saikofish](https://twitter.com/saikofish).
+
+# Docker
+
+**Building:**
+
+```bash
+docker build -t lcars:latest
```
Comment (louh) [inline README.md]: Please see comment regarding this command.

**Review by louh — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (dragonchaser) [reply README.md]: I will take a look, there have been some changes to docker since I wrote this. Haven't tried with buildx at the time.

**Comment by dragonchaser** [comment]
@louh can you take another look?

**Review by louh — APPROVED** [review]
LGTM! Thanks for the update.

**Pull request merged**